supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -447,7 +447,8 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
           privateDockerKeyAndEncryptedToken = dockerKeyAndToken,
           womOutputRuntimeExtractor = jobDescriptor.workflowDescriptor.outputRuntimeExtractor,
           adjustedSizeDisks = adjustedSizeDisks,
-          virtualPrivateCloudConfiguration = jesAttributes.virtualPrivateCloudConfiguration
+          virtualPrivateCloudConfiguration = jesAttributes.virtualPrivateCloudConfiguration,
+          retryWithDoubleMemoryKeys = jesAttributes.retryWithDoubleMemoryKeys
```

Comment:
Pro Tip: If you leave a trailing comma here and on the lines below, you can pay-it-forward so future devs won't get extra `git blame` for just adding a `,` to each list.

https://docs.scala-lang.org/sips/completed/trailing-commas.html